### DISCUSSION TITLE
[Release] When is next stable release scheduled?

### DISCUSSION THREAD

**Discussion by YairHalberstadt** [abhinavsingh/proxy.py#789] [Q&A] [answered]
Hi

I'm currently only able to use develop branch as I get version conflicts for typing-extensions in v2.3.1 which were fixed in https://github.com/abhinavsingh/proxy.py/pull/641. I'm wondering when this will be released?

Thanks for all your efforts!

**Comment by abhinavsingh** [comment]
Thank you for bringing this to my notice.  Indeed, we removed pin on `typing-extensions`.  We are setting up automated release.  It's already working for `develop` branch, see https://github.com/abhinavsingh/proxy.py/deployments/activity_log?environment=release-testpypi.  A release is made for every submitted PR.

For stable release to happen, we must merge `develop` -> `master`.  I am planning to do it soon, but haven't thought about a date.  Let's make an `rc` release sometime next week and then a final next stable release.

PS: Quite a few things have changed in the core.  I myself use `develop` in production all the time.  Things look stable, hopefully we'll make a final stable release.

**Comment by YairHalberstadt** [comment]
Thanks for the quick response!

**Comment by abhinavsingh** [comment, marked as answer]
Created PR in preparation of next release.  Ref

- https://github.com/abhinavsingh/proxy.py/pull/777

See PR description for release plan.
